R/transformCounts.R

```diff
@@ -177,11 +177,7 @@ setMethod("transformAssay", signature = c(x = "SummarizedExperiment"),
         }
         method <- match.arg(method, several.ok = FALSE)
         # Check that MARGIN is 1 or 2
-        if( !(length(MARGIN) == 1L && MARGIN %in%
-              c("samples", "features", "columns", "col", "row")) ){
-            stop("'MARGIN' must be 'samples' or 'features'.",
-                 call. = FALSE)
-        }
+        .check_MARGIN(MARGIN)
```

Comment:
.check_MARGIN gives 1 or 2 as an input. Does transformAssay work with by=1, by=2, by="column"

To harmonize, get the output value. It is easier to maintain the code when the value is either 1 or 2

MARGIN <- .check_MARGIN(MARGIN)